lib/http_server_test_fixture/mock_routes.ex

```diff
@@ -2,9 +2,9 @@ defmodule HTTPServerTestFixture.MockRoutes do
   alias HTTPServer.Request
   alias HTTPServerTestFixture.Handlers.MockGet
   alias HTTPServerTestFixture.Handlers.MockPost
-  alias HTTPServerTestFixture.Handlers.MockNotFound
+  alias HTTPServer.Handlers.NotFound
```

Comment:
```suggestion
alias HTTPServerTestFixture.Handlers.NotFound
```

trying to keep all the fixtures consistent.